lib/utils/configurationComment.mjs

```diff
@@ -37,14 +38,22 @@ export function extractConfigurationComment(
 /**
  * Tests if the given comment is a Stylelint command.
  *
- * @param {string} commentText
+ * @param {string | import('postcss').Node} textOrNode
  * @param {string} [configurationComment]
  * @returns {boolean}
  */
 export function isConfigurationComment(
-	commentText,
+	textOrNode,
 	configurationComment = DEFAULT_CONFIGURATION_COMMENT,
 ) {
+	const commentText = isString(textOrNode)
+		? textOrNode
+		: isComment(textOrNode)
+			? textOrNode.text
+			: undefined;
```

Comment:
I personally try to avoid nested ternaries, but in this case it is still fairly readable 😄

Reply:
Yeah, if this logic becomes more complex, let's decompose it. 😃